lib/Echidna/Campaign.hs

```diff
@@ -174,9 +184,27 @@ execTxOptC t = do
   res <- execTxWith vmExcept (usingCoverage $ pointCoverage (hasLens . coverage)) t
   hasLens . coverage %= unionWith union og
   grew <- (== LT) . comparing coveragePoints og <$> use (hasLens . coverage)
-  when grew $ hasLens . genDict %= gaddCalls ([t ^. call] ^.. traverse . _SolCall)
+  --when grew $ hasLens . genDict %= gaddCalls ([t ^. call] ^.. traverse . _SolCall)
```

Comment:
delete, also why are we not adding "coverage increasing" calls to the `GenDict` anymore?

Reply:
Fixed. I'm not sure the usage of coverage like that is so effective, but we can keep that line uncommented anyway.